packages/background/rpch/apply_patch.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * This is a total hack, sorry.
+ * Better fix coming later.
+ */
+const fs = require('fs');
+const path = require('path');
+const DIST_DIR = path.join(__dirname, '..', '..', '..', 'dist');
+
+function patchBackgroundJs() {
+    console.log('[RPCh] Applying patch for background.js');
+    const filePath = path.join(DIST_DIR, 'background.js');
+    const content = fs.readFileSync(filePath, 'utf-8');
+    const updatedContent = content.replace(
```

Comment:
this patch doesn't work when the build is created in production mode (minified and without source maps). You can try this by running the `make build/prod` command

Reply:
We will change to patching the minified version instead. Thanks for pointing it out